### ISSUE TITLE
Exception raised: [Errno 2] No such file or directory: '/Users/maudegull/.smh/tmp1xjDTu/64IOM0EKOG.EW_all.moog'

### ISSUE THREAD

**Issue by mgull19** [andycasey/smhr#290]
An exception was encountered using version e94df1f* on Python 2.7.14 |Anaconda custom (x86_64)| (default, Mar 12 2018, 07:39:26) [GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)]:

````python
Traceback (most recent call last):

  File "/Users/maudegull/Desktop/UROP_FREBEL/smhr/smh/gui/ui_mainwindow.py", line 463, in save_session
    self.session.save(self.session_path, overwrite=True)

  File "/Users/maudegull/iCloud Drive (Archiv)/Desktop/UROP_FREBEL/smhr/smh/session.py", line 285, in save
    tarball.add(path, arcname=os.path.basename(path))

  File "/Users/maudegull/anaconda2/lib/python2.7/tarfile.py", line 2009, in add
    tarinfo = self.gettarinfo(name, arcname)

  File "/Users/maudegull/anaconda2/lib/python2.7/tarfile.py", line 1881, in gettarinfo
    statres = os.lstat(name)

OSError: [Errno 2] No such file or directory: '/Users/maudegull/.smh/tmp1xjDTu/64IOM0EKOG.EW_all.moog'
````

Below is a screenshot from the application at the time that the exception occurred:

![screenshot](https://transfer.sh/YaKwQ/screenshot.png)

**Additional details on how this exception occurred**:
When trying to save file,

**Comment by alexji** [comment]
This is a bug I introduced when trying to keep track of the original line
lists that were imported for backup purposes. In some cases it does not
make the copy properly. However it does not affect functionality or the
saving (the error is thrown after saving all other files).

**Comment by alexji** [comment]
I think this was fixed in the most recent patch (see #307)
